### PR TITLE
[WIP] Turbinia tests

### PR DESCRIPTION
**Pull request by tomchop**
These tests will fail until https://github.com/google/turbinia/issues/293 is adressed.

### CODE REVIEW

**Comment by tomchop** [comment]
The tests are failing because the version of Turbinia that is being pulled from pypi doesn't seem to have the changes from https://github.com/google/turbinia/pull/294. This PR can be kept open and CI re-run when the changes have been pushed :)

File: tests/lib/processors/test_data/turbinia.conf [hunk 1]
```diff
@@ -0,0 +1,57 @@
+"""Turbinia test config file."""
```
Comment (rgayon) [inline tests/lib/processors/test_data/turbinia.conf]: Why is this file in lib/processors ?

File: tests/lib/processors/turbinia.py [hunk 2]
```diff
@@ -19,6 +29,139 @@ def testInitialization(self):
...
+    error_msg, is_critical = test_state.errors[0]
+    self.assertEqual(error_msg, 'Specified project turbinia-wrong-project does'
+                                ' not match Turbinia configured project '
+                                'turbinia-project. Use gcp_turbinia_import '
+                                'recipe to copy the disk into the same '
+                                'project.')
+    self.assertEqual(is_critical, True)
+
+  @mock.patch('turbinia.client.TurbiniaClient')
+  def tesWrongSetups(self, _mock_TurbiniaClient): # pylint: disable=invalid-name
```
[the remark this answers is not in the document]
Comment (tomchop) [reply tests/lib/processors/turbinia.py]: ixed

**Comment by tomchop** [comment]
Seems that I can't ignore codefactor errors. Maybe @Onager has ideas why? I'll merge the PR in the meantime, since it's just complaining about `/tmp` directories in tests.

**Pull request merged**